packages/config/config/devices/0x010f/fgwdeu.json

```diff
@@ -162,445 +145,642 @@
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1
+			"label": "Automatic Control: Dimming Step Size",
+			"$import": "~/templates/master_template.json#base_1-99_nounit"
 		},
 		{
 			"#": "155",
-			"label": "Automatic control – time of dimming step",
+			"label": "Automatic Control: Dimming Step Duration",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 1
+			"defaultValue": 1,
+			"unit": "0.01 seconds"
 		},
 		{
 			"#": "156",
-			"label": "Manual control – dimming step size",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1
+			"label": "Manual Control: Dimming Step Size",
+			"$import": "~/templates/master_template.json#base_1-99_nounit"
 		},
 		{
 			"#": "157",
-			"label": "Manual control – time of dimming step",
+			"label": "Manual Control: Dimming Step Duration",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 5
+			"defaultValue": 1,
+			"unit": "0.01 seconds"
 		},
 		{
 			"#": "158",
-			"label": "Auto-off functionality",
+			"label": "Auto-off Timer",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32767,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"unit": "seconds",
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		{
 			"#": "159",
-			"label": "Force auto-calibration",
+			"label": "Force Calibration",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 2,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Without FIBARO Bypass 2",
+					"value": 1
+				},
+				{
+					"label": "With FIBARO Bypass 2",
+					"value": 2
+				}
+			]
 		},
 		{
 			"#": "160",
-			"label": "Auto-calibration status",
+			"label": "Calibration Status",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
-			"defaultValue": 0,
-			"readOnly": true
+			"readOnly": true,
+			"options": [
+				{
+					"label": "Uncalibrated or overridden",
+					"value": 0
+				},
+				{
+					"label": "Calibrated",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "161",
-			"label": "Burnt out bulb detection",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 99,
-			"defaultValue": 0
+			"label": "Burnt Out Bulb Detection: Power Variation",
+			"$import": "~/templates/master_template.json#base_0-99_nounit",
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		{
 			"#": "162",
-			"label": "Delay of a burnt out bulb and overload detection",
+			"label": "Time Delay: Burnt Out Bulb and Overload Detection",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 5
+			"defaultValue": 5,
+			"unit": "seconds"
 		},
 		{
 			"#": "163",
-			"label": "1st button ON value sent to association groups",
+			"label": "First Button ON Value Sent to Association Groups",
```

Comment:
```suggestion
			"label": "First Button: ON Value Sent to Association Groups",
```

Reply:
But please check the templates, I think we have one for this and the next one too.